.github/workflows/build_and_test.yml

```diff
@@ -888,7 +888,13 @@ jobs:
           helm upgrade ${{ matrix.proxy_job }}       neondatabase/neon-proxy --namespace neon-proxy --install -f .github/helm-values/${{ matrix.proxy_config }}.yaml --set image.tag=${DOCKER_TAG} --wait --timeout 15m0s
           helm upgrade ${{ matrix.proxy_job }}-scram neondatabase/neon-proxy --namespace neon-proxy --install -f .github/helm-values/${{ matrix.proxy_config }}-scram.yaml --set image.tag=${DOCKER_TAG} --wait --timeout 15m0s
 
-  deploy-proxy-new:
+      - name: Re-deploy storage-broker
```

Comment:
We'd better run this only when storage_broker directory changes. Unfortunately, GH actions doesn't support such filtering out of the box on job/step level, but there is 
https://github.com/dorny/paths-filter

Feel free to add it here or leave that for a separate PR.

Also, deploy-prod-new still misses the deploy, but I'm sure this is known.

Also, I don't see much value in customizing namespace name, but that's nitpicking.

Reply:
> We'd better run this only when storage_broker directory changes.

We would need to do this also when ../libs or ../utils change, as the storage_broker seems to depend on those paths, am I right? I have a bit of a worry, it would just get confusing, as we update the same image, but would then run old version of the image. I think we probably need to have a separate discussion about how we handle this, as we don't really do that for anything else that's in the same image.

> Also, deploy-prod-new still misses the deploy, but I'm sure this is known.

Yeah, a bit work in progress.

> Also, I don't see much value in customizing namespace name, but that's nitpicking.

Namespaces are great way to separate apps and their resources. When app deployments (or environment, in case of neon-stress) are separated to their own namespaces, implementing  NetworkPolicies and RBAC becomes possible in a sane way.

Reply:
> We would need to do this also when ../libs or ../utils change

Yes, good catch.